packages/cfpb-core/src/base.less

```diff
@@ -312,12 +312,12 @@ h6,
 .lead-paragraph {
   .heading-3();
 
-  margin-top: unit((30px / @font-size), em);
-  margin-bottom: unit((15px / 18px), em);
+  margin-top: 0;
+  margin-bottom: 15px;
 
   .respond-to-max(@bp-xs-max, {
     // Use the same regular weight but reduce the sizes to h4 size
-    margin-top: unit( ( 30px / 18px ), em );
+    margin-top: 0;
```

Comment:
We don't need this, right? Since it is set on line 315. Removed in https://github.com/cfpb/design-system/pull/1789